src/config_pepperflash.c

```diff
@@ -63,7 +63,7 @@ fpp_config_get_plugin_name(void)
 const char *
 fpp_config_get_default_plugin_descr(void)
 {
-    return "Shockwave Flash 13.1 r2";
+    return "Shockwave Flash 24.0 r186";
```

Comment:
It should be `24.0 r0` here. Third number goes into `rXXX` part, while fourth is just dropped.